twingate/client_resource.go

```diff
@@ -13,12 +13,15 @@ import (
 const readResourceQueryGroupsSize = 50
 
 var (
-	ErrTooManyGroupsError        = errors.New("provider does not support more than 50 groups per resource")
-	ErrGraphqlIDIsEmpty          = errors.New("id is empty")
-	ErrGraphqlNameIsEmpty        = errors.New("name is empty")
-	ErrGraphqlConnectorIDIsEmpty = errors.New("connector id is empty")
-	ErrGraphqlNetworkIDIsEmpty   = errors.New("network id is empty")
-	ErrGraphqlNetworkNameIsEmpty = errors.New("network name is empty")
+	ErrTooManyGroupsError              = errors.New("provider does not support more than 50 groups per resource")
```

Comment:
It doesn't seem like `client_resource` is the right place for these to be as these are generic to all clients...

Reply:
Feel free to fix on a separate PR